modules/core/vault-helper.sh

```diff
@@ -14,6 +14,8 @@ readonly SCRIPT_NAME="$(basename "$0")"
 readonly MAX_RETRIES=30
 readonly SLEEP_BETWEEN_RETRIES_SEC=10
 
+TERRAFORM="${TERRAFORM:-terraform}"
```

Comment:
Never knew of this if-not-assigned-then-assign syntax :+1:

Reply:
More of the `bash-isms` LOL